### DISCUSSION TITLE
Setup guide for those who never use Ansible

### DISCUSSION THREAD

**Discussion by tinusriyanto** [HCL-TECH-SOFTWARE/connections-automation#4] [General]
Here is what I assume I will need to do in order to run this.
1. Install Asible on a control-node machine (laptop / PC) that is connected to all the servers that will host Connections.
2. Download this repo and copy to control node.
3. Download all the required installer and placed them in a folder, preferably with the folder structure in your given example.
4. Setup small HTTP server that will serve those installer. You mention Ruby, what kind of Ruby ? Is there any example ?
And I am already stuck :D.

**Comment by dejo1307** [comment]
Actually, good question :))

It is simple like this:
- Simply follow up https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#installing-ansible-with-pip (this is the most straight forward; if you are setting one of yours nodes to be also the controller, and let's say you are on CentOS for example, just do yum install ansible)
- To run simple web server from that folder where you put your files, and you installed Ruby already, just run: ruby -run -e httpd . -p 8001

I will write a small how to and add it here latest EOD tomorrow about how to do basic setup and where to pay attention, and put also examples why it is the way it is.

**Comment by dejo1307** [comment]
Check out https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/main/documentation/QUICKSTART.md @tinusriyanto I hope this helps
